lib/player.js

```diff
@@ -4,96 +4,72 @@ function Player(name, x, y, mass, keyTracker) {
   this.y    = y;
   this.mass = mass || 5;
   this.keyTracker = keyTracker;
-
-  this.movePlayer = function(xOffset, yOffset) {
-    this.x += xOffset;
-    this.y += yOffset;
-  };
-  this.moveLeft = this.movePlayer.bind(this, -1, 0);
-  this.moveRight = this.movePlayer.bind(this, 1, 0);
-  this.moveUp = this.movePlayer.bind(this, 0, -1);
-  this.moveDown = this.movePlayer.bind(this, 0, 1);
 }
 
 Player.prototype = {
-  move: function() {
-    this.movePlayer1();
-    this.movePlayer2();
-    return this;
-  },
-
-  eatFood: function(food) {
-    for(var i = 0; i < food.length; i++) {
-      var xDiff = this.x - food[i].x;
-      var yDiff = this.y - food[i].y;
-      var distance = Math.sqrt( xDiff*xDiff + yDiff*yDiff );
-      if(distance < this.mass) {
-        food.splice(i, 1);
-        this.mass += 1;
-      }
-    }
+  speed: function() {
+    if (this.mass > 50){
+      return 1;
+    } else if (this.mass > 30) {
+      return 2
+    } else if (this.mass > 20){
+      return 3
+    } else if (this.mass > 10) {
+      return 4
+    } else {
+      return 5
+    };
```

Comment:
Is it possible to make speed an attribute of the player that is adjusted when eating, instead of a function with if statements? This may be a performance increase.